pyproject.toml

```diff
@@ -70,28 +70,44 @@ before-build = "export HOMEBREW_AUTO_UPDATING=0 && brew update && brew install n
 [tool.mypy]
 ignore_missing_imports = true
 
-[tool.isort]
-line_length = 120
-skip_glob = ["*.pyi"]
+# [tool.isort]
+# line_length = 120
+# skip_glob = ["*.pyi"]
 
 [tool.ruff]
 line-length = 120
```

Comment:
This may need to be changed to 100, see https://github.com/FEniCS/dolfinx/pull/3044#discussion_r1481258501

Reply:
50 once black-style formatting is applied.